dts/arm/microchip/mec172x/mec172xnsz-pinctrl.dtsi

```diff
@@ -621,24 +617,28 @@
 		pinmux = < MCHP_XEC_PINMUX(0126, MCHP_AF1) >;
 	};
 
-	gpspi_cs_n_gpio116: gpspi_cs_n_gpio116 {
-		pinmux = < MCHP_XEC_PINMUX(0116, MCHP_AF1) >;
+	gpspi_cs_n_gpio024: gpspi_cs_n_gpio024 {
+		pinmux = < MCHP_XEC_PINMUX(024, MCHP_AF1) >;
 	};
 
-	gpspi_clk_gpio117: gpspi_clk_gpio117 {
-		pinmux = < MCHP_XEC_PINMUX(0117, MCHP_AF1) >;
+	gpspi_clk_gpio023: gpspi_clk_gpio023 {
+		pinmux = < MCHP_XEC_PINMUX(023, MCHP_AF1) >;
 	};
 
-	gpspi_io0_gpio074: gpspi_io0_gpio074 {
-		pinmux = < MCHP_XEC_PINMUX(074, MCHP_AF1) >;
+	gpspi_io0_gpio245: gpspi_io0_gpio245 {
+		pinmux = < MCHP_XEC_PINMUX(0245, MCHP_AF1) >;
 	};
 
-	gpspi_io1_gpio075: gpspi_io1_gpio075 {
-		pinmux = < MCHP_XEC_PINMUX(075, MCHP_AF1) >;
+	gpspi_io1_gpio243: gpspi_io1_gpio243 {
+		pinmux = < MCHP_XEC_PINMUX(0243, MCHP_AF1) >;
 	};
 
-	gpspi_wp_n_gpio076: gpspi_wp_n_gpio076 {
-		pinmux = < MCHP_XEC_PINMUX(076, MCHP_GPIO) >;
+	gpspi_io2_gpio034: gpspi_io2_gpio034 {
+		pinmux = < MCHP_XEC_PINMUX(034, MCHP_AF1) >;
+	};
+
+	gpspi_io3_gpio022: gpspi_io3_gpio022 {
+		pinmux = < MCHP_XEC_PINMUX(0022, MCHP_AF1) >;
 	};
```

Comment:
We are reviewing gpspi nomenclature used in the specification and confirm if this change is required.